setup.py

```diff
@@ -7,104 +7,106 @@
 import sys
 
 
-if LooseVersion(sys.version) < LooseVersion('3.7'):
+if LooseVersion(sys.version) < LooseVersion("3.7"):
     raise RuntimeError(
-        'ESPnet requires Python>=3.7, '
-        'but your Python is {}'.format(sys.version))
-if LooseVersion(pip.__version__) < LooseVersion('19'):
+        "ESPnet requires Python>=3.7, but your Python is {}".format(sys.version)
+    )
+if LooseVersion(pip.__version__) < LooseVersion("19"):
     raise RuntimeError(
-        'pip>=19.0.0 is required, but your pip is {}. '
-        'Try again after "pip install -U pip"'.format(pip.__version__))
+        "pip>=19.0.0 is required, but your pip is {}. "
+        'Try again after "pip install -U pip"'.format(pip.__version__)
+    )
 
 requirements = {
-    'install': [
+    "install": [
         # 'torch==1.0.1',  # Installation from anaconda is recommended for PyTorch
-        'chainer==6.0.0',
+        "chainer==6.0.0",
```

Comment:
we may add some comments that we'll remove chainer installation in the near future.